server/data/elite2ClientBuilder.js

```diff
@@ -49,5 +49,16 @@ module.exports = token => {
             : logger.warn(error, `Error calling elite2`),
       })
     },
+    getActiveAgenciesByType(type) {
+      return restClient.get({
+        path: `/api/agencies/type/${type}`,
+      })
+    },
+
+    getPrisonById(agencyId) {
```

Comment:
We're a bit inconsistent with what we call the id of prisons - in the future we can look at normalising this.

Reply:
Yes, I wasn't sure which name to use because in the report and elite2 its referred to as agencyId.